examples/dbfv/psi/main.go

```diff
@@ -106,17 +105,17 @@ func main() {
 	ternarySamplerMontgomery := ring.NewTernarySampler(prng, ringQP, 0.5, true)
 
 	// Create each party, and allocate the memory for all the shares that the protocols will need
-	P := make([]*party, N, N)
```

Comment:
For consistency, shouldn't the PSI example follow the same structure as the PIR one ?

Reply:
Yes we need to reduce its cyclomatic complexity too

Reply:
Done